packages/foreman/rubygem-net-scp/rubygem-net-scp.spec

```diff
@@ -5,15 +5,12 @@
 
 Summary: A pure Ruby implementation of the SCP client protocol
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.2.1
-Release: 4%{?dist}
+Version: 3.0.0
+Release: 1%{?dist}
 Group: Development/Languages
 License: MIT
 URL: http://net-ssh.rubyforge.org/scp
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-# Fix test broken due to Net::SSH 4.0+.
-# https://github.com/net-ssh/net-scp/pull/30
-Patch0: net-scp-1.2.1-Fix-compatiblity-with-net-ssh-4.0.patch
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(net-ssh)
```

Comment:
This now has a version bound. Perhaps the spec should be regenerated.

Reply:
good idea, thanks, looks better now :+1: